### PR TITLE
bugfix

### PR DESCRIPTION
**Pull request by vivekbhr**

### CODE REVIEW

File: hicexplorer/hicBuildMatrix.py [hunk 1]
```diff
@@ -14,12 +14,6 @@
 
 debug = 1
 
-# check pysam version
-from distutils.version import LooseVersion
-if LooseVersion(pysam.__version__) < LooseVersion("0.8.3"):
-    exit("\n*ERROR*\n\nVersion of pysam has to be higher than 0.8.3. Current installed version is {}\n".format(pysam.__version__))
-
-
```
Comment (fidelram) [inline hicexplorer/hicBuildMatrix.py]: Why did you remove this lines? Even though the installer checks that pysam should be higher than 0.8.3, the user could have other version because of PYTHONPATH? I remember I put those lines after some user had a problem.

Comment (vivekbhr) [inline hicexplorer/hicBuildMatrix.py]: RTD was showing build issues precisely here (and i thought it was redundant with the setup).

Although the build error still exists :(

**Comment by vivekbhr** [comment]
ok. now it build the docs locally after hicEx install. but still fails at RTD.. Reports the same error at line :
` if LooseVersion(pysam.__version__) < LooseVersion("0.8.3"): ` ... Even though I removed it. 

That means a fresh install is needed at RTD, but if i turn it on it won't install. so caught in a loop

**Comment by vivekbhr** [comment]
All right.. all green..!
Note to self : try reverting commits rather than adding new commits to revert changes. SAD!
